main.py

```diff
@@ -53,20 +53,21 @@ def github_repo_to_api(url):
 
 def github_repo_to_api_releases(url):
     """Converts a github repoository url to the api entry with the releases"""
-    url = normalize_url(url)
-    url = url.replace("https://github.com/", "https://api.github.com/repos/")
+    url = github_repo_to_api(url)
     url += "/releases"
     return url
 
 
 def normalize_url(url):
     """
-    Canonical urls be like: no slash, no file extension
+    Canonical urls be like: https, no slash, no file extension
 
     :param url:
     :return:
     """
     url = url.strip("/")
+    url = url.replace("http://", "https://")
+    url = url.replace("git://", "https://")
```

Comment:
This combined with `[a-z]+` pattern would break on anything but git and https, let's say if for some reason someone puts in ssh. I'd prefer a regex replace or a simple `"https://" + url.split("://")[1]`.

Reply:
In practice it wouldn't break since we already filter for http/https/git in the sparql-query – but you  are absolutely right that we should do this the right way. Actually I first used re.sub but changed it since regular expressions felt like a overkill – but I like your idea with split. Will do this.

Reply:
done

Reply:
Thanks!